FirebaseAuth/Sources/Auth/FIRAuth.m

```diff
@@ -1788,35 +1768,8 @@ - (BOOL)updateKeychainWithUser:(FIRUser *)user error:(NSError *_Nullable *_Nulla
   return NO;
 }
 
-/** @fn setKeychainServiceNameForApp
-    @brief Sets the keychain service name global data for the particular app.
-    @param app The Firebase app to set keychain service name for.
- */
-+ (void)setKeychainServiceNameForApp:(FIRApp *)app {
-  @synchronized(self) {
-    gKeychainServiceNameForAppName[app.name] =
-        [@"firebase_auth_" stringByAppendingString:app.options.googleAppID];
-  }
-}
-
-/** @fn keychainServiceNameForAppName:
-    @brief Gets the keychain service name global data for the particular app by name.
-    @param appName The name of the Firebase app to get keychain service name for.
- */
-+ (NSString *)keychainServiceNameForAppName:(NSString *)appName {
-  @synchronized(self) {
-    return gKeychainServiceNameForAppName[appName];
-  }
-}
-
-/** @fn deleteKeychainServiceNameForAppName:
-    @brief Deletes the keychain service name global data for the particular app by name.
-    @param appName The name of the Firebase app to delete keychain service name for.
- */
-+ (void)deleteKeychainServiceNameForAppName:(NSString *)appName {
-  @synchronized(self) {
-    [gKeychainServiceNameForAppName removeObjectForKey:appName];
-  }
++ (NSString *)keychainServiceNameForApp:(FIRApp *)app {
+  return [@"firebase_auth_" stringByAppendingString:app.options.googleAppID];;
```

Comment:
Replace with a helper method to determine the keychain service name given an app.